shark_turbine/kernel/wave/expansion.py

```diff
@@ -178,17 +179,22 @@ def compute_index(node: fx.Node) -> bool:
 def expand_graph(
     trace: CapturedTrace,
     constraints_or_scaling: Sequence[Constraint] | dict[IndexSymbol, int],
+    tile_sizes: dict[IndexSymbol, int] = None,
```

Comment:
`tile_sizes` is never checked for `None` before assigning to `dim_tile_size`, and  `dim_tile_size` == `None` will break things.

Reply:
Adding an assert in corresponding branch will be a good idea. Also, type should be `Optional[dict[IndexSymbol, int]]` in this case

Reply:
Thanks, will add that check here!